fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -210,6 +210,7 @@ NOT: 'NOT';
 NULL: 'NULL';
 NULLS: 'NULLS';
 OBSERVER: 'OBSERVER';
+OF: 'OF';
```

Comment:
OF should added to the `nonReserved` too.

Reply:
added in 6c9b4cf20